README.rst

```diff
@@ -253,3 +244,51 @@ This project is distributed under the Apache license version 2.0 (see the LICENS
 
 By submitting a pull request to this project, you agree to license your contribution under the Apache license version
 2.0 to this project.
+
+Citation
+========
+
+|DOI|
+
+Iterative, *DVC: Data Version Control - Git for Data & Models* (2020)
+`DOI:10.5281/zenodo.012345 <https://doi.org/10.5281/zenodo.3677553>`_.
```

Comment:
Again, not sure about capitalisation and exact wording

Reply:
I think you can write, `Iterative, Inc. *Data Version Control (DVC)*`. No need to say "Git for Data & Models"

Reply:
let's discuss this with @dmpetrov ... we need to get it right.

- do we need to mention org at all? Iterative?
- do we optimize for certain keywords. I would say "Git for Data" is important for us, even though Data Version Control is even more important.

but may be you are right, @andronovhopf and we should go with a clean and nice, simple version 🤔

Reply:
I don't think it's terribly important to optimize for keywords in this context since Zenodo doesn't look built for discovering new projects; just tagging the repo with a DOI. not sure though.

Reply:
I think @shcheklein was referring to keywords (buzzwords) to use in the recommended citaion

Reply:
ah gotcha